javascript/employee.js

```diff
@@ -0,0 +1,840 @@
+    })
+}
+
+// hiding  and opening the advance filter when we add click on filter icon
+var filter = true;
+function hideAdvanceFilter() {
+    const advanceFilter = document.querySelector(".advance-filter-section");
+    const table = document.querySelector(".table-container");
+    const tableBody = document.querySelector(".table-div");
+    if (filter) {
+        advanceFilter.style.display = "none";
+        table.style.height = "350px";
+        tableBody.style.height = "270px";
+    }
+    else {
+        advanceFilter.style.display = "flex";
+        table.style.height = "292px";
+        tableBody.style.height = "235px";
+    }
+    filter = !filter;
+}
+
+// display searched values on clicking enter
+function pressEnter(event) {
+    var input = document.getElementById("searchBar").value.toUpperCase();
+    if (event.keyCode == 13 && input.length >= 3) {
+        searchTable(input);
+    }
+    else {
+        let tbody = document.getElementById("tableBody");
+        tbody.innerHTML = ' ';
+        createTable(employee);
+        attachStatusOptionListners();
+    }
+}
+
+function searchTable(input) {
+    var tableData = document.querySelector(".table");
+    var tr = Array.from(tableData.getElementsByClassName("row"));
+    let tableRow = [];
+    tr.forEach(r => {
+        let rowData = [];
+        let col = r.querySelectorAll("td");
+        col.forEach(cell => {
+            if (cell.cellIndex === 1) {
+                let image = cell.querySelector("img");
+                rowData["employeeImg"] = image.src;
+                let userdata = cell.innerText.trim();
+                let [name, email] = userdata.split("\n");
+                rowData["Name"] = name;
+                rowData["mailId"] = email;
+            }
+            else if (cell.cellIndex === 2) {
+                rowData["location"] = cell.innerText.trim();
+            }
+            else if (cell.cellIndex === 3) {
+                rowData["department"] = cell.innerText.trim();
+            }
+            else if (cell.cellIndex === 4) {
+                rowData["role"] = cell.innerText.trim();
+            }
+            else if (cell.cellIndex === 5) {
+                rowData["empNo"] = cell.innerText.trim();
+            }
+            else if (cell.cellIndex === 6) {
+                rowData["status"] = cell.innerText.trim();
+            }
+            else if (cell.cellIndex === 7) {
+                rowData["dateofJoin"] = cell.innerText.trim();
+            }
+        });
+        tableRow.push(rowData);
+    })
+    let arr = [];
+    tableRow.forEach(r => {
+        let txtValue = r.Name;
+        if (txtValue.toUpperCase().indexOf(input) > -1) {
+            arr.push(r);
+        }
+    })
+    let tbody = document.getElementById("tableBody");
+    tbody.innerHTML = ' ';
+    createTable(arr);
+    attachStatusOptionListners();
+}
+
+//  alphabet filter 
+applyalphabetFilter();
+let btn = "";
+function applyalphabetFilter() {
```

Comment:
Use Camel case in function name and try to reduce if and else statements in this function.